render-wasm/docs/serialization.md

```diff
@@ -208,6 +216,28 @@ Shadow styles are serialized as `u8`:
 | 6     | Stretch       |
 | \_    | error         |
 
+### Align Self
+
+| Value | Field   |
+| ----- | ------- |
+| 0     | Auto    |
+| 1     | Start   |
+| 2     | End     |
+| 3     | Center  |
+| 4     | Stretch |
+| \_    | error   |
+
+### Justify Self
+
+| Value | Field   |
+| ----- | ------- |
+| 0     | Auto    |
+| 1     | Start   |
+| 2     | End     |
+| 3     | Center  |
+| 4     | Stretch |
+| \_    | error   |
+
 ### Wrap type
```

Comment:
I noticed we're not adding here `GapType` and `PaddingType`, is it on purpose cause it's not needed?

Reply:
It's not needed, i'm always sending the 4 values of gap and the two values of padding (and the same with margins) so we don't really need the "type". To tell you the truth the type is something we only use in the right sidebar but it's not necessary for the geometry.

Reply:
understood 👍🏽 thanks for the explanation!